airbyte-integrations/connectors/destination-iceberg/Dockerfile

```diff
@@ -21,5 +21,5 @@ ENV JAVA_OPTS="--add-opens java.base/java.lang=ALL-UNNAMED \
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=0.1.0
+LABEL io.airbyte.version=0.2.0
```

Comment:
@nastra why 0.1.0 to 0.2.0 and not 0.1.1? Any change break changes you're doing?

Reply:
increasing the patch versions seems not appropriate to me because we're increasing Iceberg from 1.0.0 to 1.1.0, which had a few bigger changes that went in (incl deprecated API cleanups and behavioral changes)

Reply:
But what is the impact for Airbyte's usage of the connector?

Reply:
I've updated this to 0.1.1, I guess from Airbyte's POV it's probably ok to just increase the patch version here